lib/main.dart

```diff
@@ -11,103 +12,11 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Flutter Demo',
       theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
+        primarySwatch: Colors.teal,
       ),
-      home: MyHomePage(title: 'Flutter Demo Home Page'),
```

Comment:
the same also for this title

Reply:
Done ✔️ (This code is not up to date) 
I think you are commenting wrong branch.

![image](https://user-images.githubusercontent.com/46793415/132092726-5fed8c3a-a8ac-4004-abf6-eb1eae4a319b.png)

Reply:
ok you're right